metrics_proto.go

```diff
@@ -53,9 +53,8 @@ var percentileLabelKey = &metricspb.LabelKey{
 type metricProtoPayload struct {
 	node             *commonpb.Node
 	resource         *resourcepb.Resource
-	metric           *metricspb.Metric
+	metric           []*metricspb.Metric
```

Comment:
Nit: would it be better to keep one `metric` per `payload` and have multiple `payload`s for `Summary` type?

/cc @odeke-em any reason we chose one `metric` per `payload` at first?

Reply:
Also consider renaming this to `metrics` if we keep it as a slice.